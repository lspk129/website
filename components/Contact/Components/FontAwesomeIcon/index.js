import styled from 'styled-components';

const FontAwesomeIcon = styled.i`
  color: ${props => props.theme.main};
`;

export default FontAwesomeIcon;
