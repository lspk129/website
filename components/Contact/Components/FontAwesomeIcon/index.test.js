import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import FontAwesomeIcon from './';

describe('FontAwesomeIcon', () => {
  const shallowWrapper = shallow(<FontAwesomeIcon />);

  it('renders correctly', () => {
    const wrapper = renderer.create(<FontAwesomeIcon />);
    const tree = wrapper.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders i tag', () => {
    expect(shallowWrapper).toBePresent();
    expect(shallowWrapper).toHaveTagName('i');
  });
});
