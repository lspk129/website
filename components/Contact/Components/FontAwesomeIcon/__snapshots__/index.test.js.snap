// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FontAwesomeIcon renders correctly 1`] = `
<i
  className="sc-bdVaJa uNRti"
/>
`;
